main/checker-node.js

```diff
@@ -16,16 +16,16 @@ const { parseEther } = require('ethers/lib/utils')
 const { once } = require('node:events')
 const { format } = require('node:util')
 
-const log = require('electron-log').scope('core')
+const log = require('electron-log').scope('checker')
```

Comment:
Since you rename the file from "core" to "checker-node", I think the logging scope should follow the same pattern.

```suggestion
const log = require('electron-log').scope('checker-node')
```

Thoughts?

Reply:
Done